### PR TITLE
LPS-193478 | Improvement | Imp/Exp task

### PR DESCRIPTION
**Pull request by JhossephG**
https://liferay.atlassian.net/browse/LPS-193478

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:stop

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase [hunk 1]
```diff
@@ -529,17 +529,27 @@ definition {
...
+
+		task ("Then the action is present on the object") {
+			ObjectPortlet.selectCustomObject(label = "Imported Object With Action");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "CanImportObjectWithAction");
+		}
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:552]: Hey @JhossephG, looks good! Just a couple of feedback:

Your action here is importing the object, so any step that does that will be under `When`. All assertions after the action will be under `Then`. If there are no requirements to import an object, then I think it's ok to skip the `Given`. Let me know what you think! Thank you!

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Sorry for the delay, @JhossephG!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase (hunk 1 above)
Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase:552]: Make sense @patriciajeanperez, I'll update the tasks!

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:forward

**Pull request closed without merging**